src/dropdown/dropdown.scss

```diff
@@ -213,23 +213,59 @@ $height-small: 30px !default;
   }
 
   &.Select--multi {
+    height: auto;
```

Comment:
My fault, this is not true... `size` it's working, but there's some UI defect if `size='small'`

![](https://files.slack.com/files-pri/T025Q7PPH-F4KRSGJR1/pasted_image_at_2017_03_17_14_55.png)